packages/@cdktf/provider-generator/lib/get/generator/module-generator.ts

```diff
@@ -166,6 +166,9 @@ function parseType(type: string) {
   if (type === "map") {
     return "{ [key: string]: string }";
   }
+  if (type === "tuple") {
+    return "tuple";
```

Comment:
`string[]` would be the best match this case. Using a raw `tuple` for a type is mostly from older Terraform versions which didn't have the same typing information available.